source/fundamentals/builders/aggregation-expression-operations.txt

```diff
@@ -1045,24 +1029,26 @@ You can perform a map operation on a value of either type ``MqlMap`` or
    * - `unset() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlMap.html#unset(com.mongodb.client.model.mql.MqlString)>`__
      - *No corresponding operator*
 
-Suppose you have a collection of theater data where each document represents
-an individual theater. One of the fields in that document is a map of the
-ticket prices for different age groups. You want to find all the theaters that
-have a rate for a senior age group.
+Suppose you have a collection of warehouse data where each document
+represents an individual warehouse. Each document contains a field that
+is a map of the quantities for individual item IDs. You want to
+represent this map as an array to prepare the data for future array
+operations.
 
-The ``has()`` method determines whether the ``rates`` map has a ``senior``
-field.
+The ``entries()`` method returns the map entries in the ``inventory``
+field as an array. This example stores the result as a new
+``inventoryArray`` field using the ``project()`` method.
 
 The following code shows the pipeline for this aggregation:
 
 .. code-block:: java
    :copyable: true
 
-   asList(match(
-         expr(current()
-               .getMap("rates")
-               .has(of("senior")))
-   ));
+   var inventory = current().getMap("inventory");
+
+   asList(project(fields(
+       computed("inventoryArray", inventory.entries())
```

Comment:
How about something like

`warehouseInventory.entries().sum(warehouse -> warehouse.getValue())`

to get the sum of all inventory in the warehouse? (I am assuming that it maps from warehouse ids to integers, and the integers count the inventory).

This would demonstrate usage of the entry type.

Reply:
changed example to use minN.